Unable to build on Windows 11?

Attempted to run with the recommended usage of `cargo install rascii_art`. However, code errors out at `expanduser` crate. Attached error message is included below:

```rust
   ...Other Packages/Crates
   Compiling serde v1.0.183                                                                                             
   Compiling parking_lot v0.12.1
   Compiling ahash v0.7.6
   Compiling time v0.1.45
   Compiling dirs v1.0.5                                                                                                
   Compiling pwd v1.4.0
   Compiling byteorder v1.4.3                                                                                           
   Compiling expanduser v1.2.2
   Compiling chrono v0.4.26
error[E0432]: unresolved import `pwd::Passwd`
  --> C:\Users\MyUser\.cargo\registry\src\index.crates.io-6f17d22bba15001f\expanduser-1.2.2\src\lib.rs:10:5
   |
10 | use pwd::Passwd;
   |     ^^^^^^^^^^^ no `Passwd` in the root

For more information about this error, try `rustc --explain E0432`.
error: could not compile `expanduser` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
```

Is there a suggested fix for this? Thanks again!

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 name = "rascii_art"
 version = "0.4.4"
-authors = [ "KoBruhh", "RGBCube" ]
+authors = [ "UTFeight", "RGBCube" ]
 description = "Advanced ASCII Art Generator"
 repository = "https://github.com/UTFeight/RASCII"
 keywords = [ "ascii" ]

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "rascii_art"
-version = "0.4.2"
+version = "0.4.3"
 authors = [ "KoBruhh", "RGBCube" ]
 description = "Advanced ASCII Art Generator"
 repository = "https://github.com/UTFeight/RASCII"
@@ -10,8 +10,8 @@ edition = "2021"
 
 [dependencies]
 ansi_term = "0.12.1"
+cfg-if = "1.0.0"
 clap = { version = "4.3.0", features = [ "derive" ] }
-expanduser = "1.2.2"
 image = "0.24.6"
 unicode-segmentation = "1.10.1"
 

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "rascii_art"
-version = "0.4.3"
+version = "0.4.4"
 authors = [ "KoBruhh", "RGBCube" ]
 description = "Advanced ASCII Art Generator"
 repository = "https://github.com/UTFeight/RASCII"

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -26,23 +26,18 @@ mod gif_renderer;
 mod image_renderer;
 mod renderer;
 
-use std::{
-    io,
-    path::Path,
-};
-use expanduser::expanduser;
 use image::DynamicImage;
 use image_renderer::ImageRenderer;
 pub use renderer::RenderOptions;
 use renderer::Renderer;
+use std::{io, path::Path};
 
 pub fn render<P: AsRef<Path> + AsRef<str>>(
     path: P,
     to: &mut impl io::Write,
     options: &RenderOptions<'_>,
 ) -> image::ImageResult<()> {
-    let absolute_path = expanduser(&path)?;
-    let image = &image::open(absolute_path)?;
+    let image = &image::open(path)?;
     render_image(image, to, &options)
 }
 
@@ -61,8 +56,7 @@ pub fn render_to<P: AsRef<Path> + AsRef<str>>(
     buffer: &mut String,
     options: &RenderOptions<'_>,
 ) -> image::ImageResult<()> {
-    let absolute_path = expanduser(&path)?;
-    let image = &image::open(absolute_path)?;
+    let image = &image::open(path)?;
     let renderer = ImageRenderer::new(image, options);
     renderer.render_to(buffer)?;
     Ok(())

